stable_diffusion/stable_diffusion/model_io.py

```diff
@@ -80,8 +97,9 @@ def map_unet_weights(key, value):
 
     if len(value.shape) == 4:
         value = value.transpose(0, 2, 3, 1)
+        value = value.reshape(-1).reshape(value.shape)
```

Comment:
Interesting.. the intention is to make the value row contiguous. Just curious did you notice it was faster?

Not suggesting a change here.. but I wonder if there is a cleaner / more general solution to this. Probably doesn't come up often except when using convolutions..

Reply:
I didn't check but it must be as it will keep being copied into a temporary just before the matmul (when conv is done with a matmul) or conv. I hate this but I don't think there is any way around knowing things about the underlying implementation.